microshift_running_apps/microshift-operators-olm.adoc

```diff
@@ -0,0 +1,64 @@
+
+toc::[]
+
+OLM is used in {microshift-short} for installing and running optional link:https://access.redhat.com/documentation/en-us/openshift_container_platform/{ocp-version}/html/architecture/control-plane#olm-operators_control-plane[add-on Operators].
+
+* Cluster Operators as applied in {ocp} are not used in {microshift-short}.
+* You can add your own catalogs for the Operators you want to use with your applications.
+* You must use the CLI to conduct OLM activities with {microshift-short}. The console and OperatorHub GUIs are not available.
+* You must configure the system to use a mirror by updating your CRI-O configuration directly because `ImageContentSourcePolicy` is not supported in {microshift-short}. See link:https://access.redhat.com/documentation/en-us/red_hat_build_of_microshift/{product-version}/html/installing/microshift-deploy-with-mirror-registry#microshift-configuring-hosts-for-mirror_microshift-deployment-mirror[Configuring hosts for mirror registry access] for details.
+* Install the link:https://access.redhat.com/documentation/en-us/openshift_container_platform/{ocp-version}/html/cli_tools/opm-cli#cli-opm-install[`opm` CLI]
+
+[IMPORTANT]
+====
+Before using an Operator, verify with the provider whether the Operator is supported on {product-title}.
+====
+
+This section of the docs is definitely referring to adding olm based operators to a disconnected cluster. I don't have a strong opinion about whether we need an entirely separate copy of the documentation, but the procedure to actually do the mirroring is definitely what we point folks to regardless of whether or not they'll have access to the openshift console. But yes, we should probably firm up our information around how to mirror images without ICSPs if we're not expecting users who already interact with microshift to know how to modify the container runtime bits directly.
+
+[id="microshift-adding-OLM-Operators-to-disconnected-cluster"]
+== Adding OLM-based Operators to a disconnected cluster
+For {microshift-short} clusters that are installed on restricted networks, also known as _disconnected clusters_, Operator Lifecycle Manager (OLM) by default cannot access sources hosted on remote registries because those remote sources require full internet connectivity.
+
+You can enable your cluster to use OLM-based Operators in a restricted network by using a workstation that has full internet access. The workstation is used to prepare local mirrors of the remote sources, and push the content to a mirror registry. See xref:../microshift_install/microshift-deploy-with-mirror-registry.adoc#microshift-deploy-with-mirror-registry[Mirroring container images for disconnected installations] and link:https://access.redhat.com/documentation/en-us/red_hat_build_of_microshift/{product-version}/html/installing/microshift-deploy-with-mirror-registry#microshift-configuring-hosts-for-mirror_microshift-deployment-mirror[Configuring hosts for mirror registry access].
```

Comment:
@ShaunaDiaz the reference Mirroring container images for disconnected installations is for microshift installation image mirror. it is not for olm-based operator index and bundle image mirror. for this part, bette to reference to the discussion  https://github.com/openshift/openshift-docs/pull/70128#discussion_r1463620564 and following discussion.

so, should remove this link here.


if there is olm component installation section later (@kevinrizza mention ithttps://github.com/openshift/openshift-docs/pull/70128#pullrequestreview-1828081481), could refer to this link with modification.
why modify this link: olm component payload is not in microshift-release-info*.noarch.rpm, and olm rpm, so need to modify the content of this link to indicate it so that the correct olm payload image is got.